main.tf

```diff
@@ -226,100 +226,3 @@ resource "aws_autoscaling_group" "container_instance" {
 resource "aws_ecs_cluster" "container_instance" {
```

Comment:
The [`aws_autoscaling_policy`](https://www.terraform.io/docs/providers/aws/r/autoscaling_policy.html) documentation suggests that we remove `desired_capacity` from the `aws_autoscaling_group` resource when using auto scaling. That makes sense because when it is present, any Terraform plan/apply cycles reset it.

Unfortunately, removing it from the `aws_autoscaling_group` resource means removing it from the module too. Seems problematic to remove it and auto scaling support.

I feel like the options are to:

1. Remove `desired_capacity` and add auto scaling support back
2. Keep auto scaling out, leave `desired_capacity` in, and play games to feed back the current `desired_capacity` via `-var` in `infra`

Reply:
To summarize the outcome from standup today:

- We want to keep auto scaling out
- We want to keep `desired_capacity` in
- We want to reevaluate things when [Terraform 0.12](https://www.hashicorp.com/blog/terraform-0-12-conditional-operator-improvements) comes out because it promises handling of a `null` `desired_capacity`